Source/JavaScriptCore/wasm/WasmAirIRGenerator64.cpp

```diff
@@ -2629,6 +2634,34 @@ auto AirIRGenerator64::addI64Or(ExpressionType arg0, ExpressionType arg1, Expres
     return { };
 }
 
+auto AirIRGenerator64::addI64ExtendSI32(ExpressionType arg0, ExpressionType& result) -> PartialResult
+{
+    result = g64();
+    append(SignExtend32To64, arg0, result);
+    return { };
+}
```

Comment:
What's the difference between `AirIRGenerator64::addI64ExtendSI32` and `AirIRGenerator64::addI64Extend32S`?  What does `SI32` mean?

Reply:
Parameter is i64 (SExt32(Trunc(input)) or i32 (SExt32(input)). And in both cases, in Air level, it is the same.